Mathlib/Data/Fin/Basic.lean

```diff
@@ -1191,20 +1191,12 @@ instance instInvolutiveNeg (n : ℕ) : InvolutiveNeg (Fin n) where
 #align fin.involutive_neg Fin.instInvolutiveNeg
 
 /-- Note this is more general than `Fin.addCommGroup` as it applies (vacuously) to `Fin 0` too. -/
-instance instIsCancelAdd (n : ℕ) : IsCancelAdd (Fin n) where
```

Comment:
What's the link between `IsCancelAdd` and your new typeclass? Do we have the instance that one implies the other?